src/redux/reducers/projects/projectsLoaded.js

```diff
@@ -5,13 +5,21 @@ const projectsLoaded = (state, action) => {
   projects.forEach((project) => {
     newProjects[project.uuid] = project;
   });
+
+  let activeProjectId = state.meta.activeProjectUuid;
+
+  // If the current active project no longer exists, change it
+  if (!Object.keys(state).includes(activeProjectId)) {
+    activeProjectId = projects[0]?.uuid;
+  }
```

Comment:
How is this currently handled? If the project is deleted, that means `activeProjectUuid: projects[0]?.uuid` is `undefined` right?

Reply:
no, `projects` is the updated list of new projects that we received from the api